qt/widgets/common/src/FunctionBrowser.cpp

```diff
@@ -744,7 +744,21 @@ QStringList FunctionBrowser::getGlobalParameters() const {
   }
   return out;
 }
-
+/**
+* Get a list of names of global parameters
+*/
+void FunctionBrowser::setGlobalParameters(QStringList &globals) {
+  for (auto propIt = m_properties.begin(); propIt != m_properties.end();
+       ++propIt) {
```

Comment:
Could you do a range-based loop here e.g. `for( auto &property : m_properties)`?

Reply:
not without considerable changes.

Reply:
Why not?